### PR TITLE
adding capabilities to package.json

### PR DESCRIPTION
**Pull request by sbatten**
@sandy081 please take a look at this and if possible help out with the configurations

### CODE REVIEW

File: extensions/npm/package.json [hunk 1]
```diff
@@ -44,9 +44,11 @@
     "workspaceContains:package.json",
     "onView:npm"
   ],
-  "workspaceTrust": {
-    "request": "onDemand",
-    "description": "This extension calls the `tasks.executeTask()` API, which requires trust to run."
+  "capabilities": {
+    "untrustedWorkspaces": {
+      "supported": "limited",
+      "description": "This extension calls the `tasks.executeTask()` API, which requires trust to run."
```
Comment (lszomoru) [inline extensions/npm/package.json:50]: Localise string.

File: extensions/php-language-features/package.json [hunk 2]
```diff
@@ -17,9 +17,11 @@
   "categories": [
     "Programming Languages"
   ],
-  "workspaceTrust": {
-    "request": "onDemand",
-    "description": "The extension requires workspace trust when the `php.validate.executablePath` setting will load a version of PHP in the workspace."
+  "capabilities": {
+    "untrustedWorkspaces": {
+      "supported": "limited",
+      "description": "The extension requires workspace trust when the `php.validate.executablePath` setting will load a version of PHP in the workspace."
```
Comment (lszomoru) [inline extensions/php-language-features/package.json:23]: Localise string.

**Review by lszomoru — APPROVED** [review]
Apart from the two localisation issues, LGTM.

**Comment by sandy081** [comment]
I am really sorry that I did not see this PR in the morning and went to adopt new shape for virtual workspaces and untrusted workspaces and pushed to main. I will see if I can resolve the conflicts and merge this PR.

**Review by sandy081 — APPROVED** [review]
LGTM

**Comment by sandy081** [comment]
Auto merged

**Pull request merged**